packages/stark-build/config/tslint.json

```diff
@@ -1,7 +1,13 @@
 {
-  "extends": ["tslint:latest", "tslint-config-prettier", "codelyzer"],
+  // FIXME: presets moved to packages/stark-core/tslint.json temporarily due to this issue: https://github.com/palantir/tslint/issues/3874
   "defaultSeverity": "error",
   "rules": {
+    // sonarts rules
```

Comment:
Can you create an issue to track these and enable them later on?

Reply:
I've created this issue: #360